manifests/init.pp

```diff
@@ -92,12 +92,12 @@
       $default_service_name   = 'syslog'
       $default_sysconfig_path = '/etc/sysconfig/syslog'
       $default_pid_file       = '/var/run/rsyslogd.pid'
-      case $::lsbmajdistrelease {
+      case $::operatingsystemmajrelease {
```

Comment:
I'm not sure this works on Suse, if you are only able to test EL, then only submit changes for that. If you do have access to Suse 11, let me know.